ntbs-ui-tests/Steps/Steps.cs

```diff
@@ -43,19 +43,39 @@ public void GivenIAmOnPage(string pageName)
         [When(@"I enter (.*) into '(.*)'")]
         public void WhenIEnterValueIntoFieldWithId(string value, string elementId)
         {
+            FindById(elementId).Click();
             FindById(elementId).SendKeys(value);
         }
 
+        [When(@"I wait")]
+        public void WhenIWait()
+        {
+            Thread.Sleep(2000);

```

Comment:
Yayikes, was 1000 not enough? It'll really start adding up to the jenkins time i'm afraid :(

Reply:
No it's actually fine with 1000 even when running in browser :)